Impartner community pack context fix

edik24 opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

Shellyber reviewed (changes requested):
@edik24 
You'll need to update the yml file, readme file, add release notes and bump the pack version as well.

edik24 commented:
> @edik24 You'll need to update the yml file, readme file, add release notes and bump the pack version as well.

actually the yml and readme are ok, the problem was with the context I prepare.
added release notes and updated version

Shellyber commented on Packs/Impartner/ReleaseNotes/1_0_1.md:
```diff
@@ -0,0 +1,6 @@
+
+##### Impartner
+
+- changed path for context argument "assigned PST engineer"
```
```suggestion
- Fixed an issue where the context path of *assigned PST engineer* was incorrect in **impartner-get-account-id** command.
```

Shellyber reviewed (approved):
nice
